src/client/pfs.go

```diff
@@ -762,6 +762,46 @@ func (c APIClient) Compact() error {
 	return err
 }
 
+type GetFileClient interface {
+	GetFile(repoName, commitID, path string, offset, size int64, writer io.Writer) error
+
+	//TODO(kdelga): Figure out if we need these.
```

Comment:
Not entirely sure what the use case of these two methods are and thus am unclear on whether they need to be implemented for the streaming use case right now. Since I've got this working without, I'm assuming it's best to just leave them out for now?

Reply:
They're used throughout our codebase (for non-batched reads) when you need a slightly different interface. Implementing the most basic one for now totally makes sense. Other ones will likely be added as they're needed in specific cases in our code.